src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString
+    .split(';')
+    .filter(item => item.trim() !== '');
+
+  const updatedSourceArray = sourceArray.map(item => {
+    const [ property, value ] = item.split(':');
+
+    return [property.trim(), value.trim()];
+  });
+
+  const sourceObject = {};
```

Comment:
Let's try to use more meaningful names
![image](https://github.com/mate-academy/js_style-to-object/assets/96409563/76dc5c30-5b35-40f7-9823-ea167c8c33f3)